core/client/components/gh-trim-focus-input.js

```diff
@@ -5,16 +5,20 @@ var TrimFocusInput = Ember.TextField.extend({
     attributeBindings: ['autofocus'],
 
     autofocus: Ember.computed(function () {
```

Comment:
I'm thinking no because autofocus seems like it should be determined once when the component is rendered and that's it. 

Are you thinking of a case where it would be useful? (to be honest, I'm not even sure if adding the autofocus attribute after the fact would trigger the browser to do anything...)